app/models/transfer.rb

```diff
@@ -0,0 +1,107 @@
+class Transfer
+  include ActiveModel::Validations
+
+  attr_accessor :amount_cents, :amount, :comission_cents, :comission, :comment,
+    :bank_account, :bank_account_id, :reference_id,
+    :inc_transaction, :out_transaction
+
+  validates :amount, presence: :true, numericality: { greater_than: 0 }
+  validates :comission, numericality: { greater_than: 0 }
+  validates_presence_of :bank_account_id
+  validates :reference_id, presence: true
+  validate :transfer_amount
+  validate :transfer_currency
+  validate :transfer_account
+
+  def initialize(attributes = {})
+    attributes.each do |name, value|
+      send("#{name}=", value)
+    end
+  end
+
+  def save
+    if valid?
+      @out_transaction = Transaction.create(
+        amount_cents: @amount_cents + @comission_cents,
+        bank_account_id: @bank_account_id, comment: form_comment(@comment),
+        category_id: Category.find_or_create_by(
+          Category::CATEGORY_BANK_EXPENSE_PARAMS).id)
+
+      @inc_transaction = Transaction.create(
+        amount_cents: @amount_cents,
+        bank_account_id: @reference_id, comment: form_comment(@comment),
+        category_id: Category.find_or_create_by(
+          Category::CATEGORY_BANK_INCOME_PARAMS).id)
+
+      if (@out_transaction.errors.any? || @inc_transaction.errors.any?)
```

Comment:
@Gataniel it's better use:
```rb
@out_transaction = Transaction.new(...)
@inc_transaction = Transaction.new(...)

if !@out_transaction.save || !@inc_transaction.save
   ... # TODO: collect errors
   return false
else
  ...
  return true
```
because save may be false without producing any errors.